neuro_san/service/http/handlers/health_check_handler.py

```diff
@@ -49,14 +58,19 @@ async def get(self):
         """
 
         try:
-            versions: Dict[str, Any] = self.determine_versions()
-            result_dict: Dict[str, Any] = {
-                "service": "neuro-san agents",
-                "status": "healthy",
-                "versions": versions
-            }
-            self.set_header("Content-Type", "application/json")
-            self.write(result_dict)
+            if self.status:
+                versions: Dict[str, Any] = self.determine_versions()
+                result_dict: Dict[str, Any] = {
+                    "service": "neuro-san agents",
```

Comment:
Do we really want to name the service here as a constant?
Should probably use AGENT_SERVER_NAME env var instead.  See the Dockerfile.